test/system/application-scans.test.ts

```diff
@@ -69,9 +69,9 @@ test("scanning a container image with 2 applications", async (t) => {
 
   t.ok(pluginResult.scannedProjects[0].meta, "os scan meta is not falsy");
   t.same(
-    pluginResult.scannedProjects[0].meta,
-    pluginResult.scannedProjects[1].meta,
-    "os scan meta and app meta are identical",
+    pluginResult.scannedProjects[0].meta.platform,
+    "linux/amd64",
+    "os scan meta includes platform information",
   );
```

Comment:
I think this was comparing that the `imageName` is contained in the meta, is it possible to also do this check?

```typescript
t.same(
  pluginResult.scannedProjects[0].meta.imageName,
  pluginResult.scannedProjects[1].meta.imageName,
  "os scan meta matches app scan meta imageName", // or something like this
);
```

Reply:
Sure think, makes sense! Done